tests/Io/SenderTest.php

```diff
@@ -106,28 +112,28 @@ public function provideRequestProtocolVersion()
      */
     public function testRequestProtocolVersion(Request $Request, $method, $uri, $headers, $protocolVersion)
     {
-        $http = $this->getMock(
+        $http = $this->getMockBuilder(
```

Comment:
@clue What should we do here? `createPartialMock` does not work for `PHPUnit 4`.

Reply:
Not sure I follow, can you elaborate what the issue is? The `getMock()` method does in fact work just fine with PHPUnit 4 or am I missing something?

Reply:
If I substitute `getMock()` to `getMockBuilder()->getMock()` for PHPUnit 5 and 6 compatibility, is didn't work. But, there's this `createParticalMock()` that solves this issue, buts it doesn't existis in `PHPUnit 4`

Reply:
Interesting! How about adding a check à la `if(phpunit 4) { dothis(); } else { dothat(); }` here?